docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -41,6 +41,28 @@ The `dagster-dbt` library installs both `dbt-core` and `dagster` as dependencies
 
 ---
 
+## Wrap your dbt project with Dagster
+
+<TabGroup>
+<TabItem name="Option 1: Create a brand new Dagster project">
+
+### Option 1: Create a brand new Dagster project
```

Comment:
Just to remove a word

```suggestion
<TabItem name="Option 1: Create a new Dagster project">

### Option 1: Create a new Dagster project
```

Reply:
Done in [6e2bb9f](https://github.com/dagster-io/dagster/pull/21816/commits/6e2bb9f11e758e37394c532832d1c995590403bb)